README.md

```diff
@@ -1,74 +1,37 @@
-# Project Name
+# Building-Blocks---Enumerables
 
-> One paragraph statement about the project.
+Enumerables
 
-![screenshot](./app_screenshot.png)
+Code on Ruby
 
```

Comment:
Improve your Readme file. It should be descriptive of the project. You are using the correct template but now you need to fill it with information about your project. You can add details about what you did in this project, why you did it, how you did it...Check this [link](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243642/) for more info.

Reply:
HI, I have added details on readme as per you suggestion.